openstack/data_source_openstack_blockstorage_volume_v2.go

```diff
@@ -0,0 +1,119 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+
```

Comment:
remove unused whitespace / extra lines

Reply:
They are a part of `go fmt` I believe. Separates `std`, `external` and `local` imports.